build.gradle

```diff
@@ -14,20 +14,20 @@ allprojects {
     }
 }
 
-// We compile the library using Java 1.6 compatibility
+// We compile the library using Java 1.7 compatibility
```

Comment:
Do we need to use 1.7 here now? Is compiling to 1.6 not supported anymore?

Reply:
Yes Gradle minimal support is now 1.7

Reply:
![image](https://user-images.githubusercontent.com/60937022/151630850-fb33fefa-be55-4b46-89d1-eac4e9f6affd.png)